src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -352,16 +396,15 @@ private HearingEntity updateHearingStatusAndVersionNumber(Long hearingId, String
             if (null != hearingEntity.getCaseHearingRequest()) {
                 log.info("CHANGING: version number : {}", hearingEntity.getCaseHearingRequest().getVersionNumber());
                 hearingEntity.getCaseHearingRequest().setVersionNumber(
-                    hearingEntity.getCaseHearingRequest().getVersionNumber() + 1);
+                    hearingEntity.getCaseHearingRequest().getVersionNumber() + VERSION_NUMBER_TO_UPDATE);
```

Comment:
Interested in this new variable VERSION_NUMBER_TO_UPDATE

Reply:
just to avoid hard code variable  , nothing much :-)